.github/workflows/ci.yml

```
@@ -95,19 +95,35 @@ jobs:
       - if: steps.detect.outputs.present == 'true'
         uses: dtolnay/rust-toolchain@stable
         with:
-          components: llvm-tools-preview
+          components: llvm-tools-preview, clippy
 
       - if: steps.detect.outputs.present == 'true'
-        uses: taiki-e/install-action@cargo-llvm-cov
+        name: build
+        run: cargo build --workspace --locked
 
       - if: steps.detect.outputs.present == 'true'
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)